StRoot/StBFChain/StBFChain.cxx

```diff
@@ -1028,21 +1029,7 @@ Int_t StBFChain::Init() {
 	  if (GetOption("VmcGeo")) {
 	    path  = "./StarDb/VmcGeo:$STAR/StarDb/VmcGeo";
 	  }
-	  TString geom("Geometry.");
-	  geom +=  DbAlias[i].geometry;
-	  geom += ".C";
-	  Char_t *file = gSystem->Which(path.Data(),geom,kReadPermission);
-	  if (file) {
-	    LOG_INFO << "StBFChain::Init force load of " << file << endm;
-	    TString command = ".L "; command += file;
-	    gInterpreter->ProcessLine(command);
-	    gInterpreter->Calc("CreateTable()");
-	    command.ReplaceAll(".L ",".U ");
-	    gInterpreter->ProcessLine(command);
-	    delete [] file;
-	  } else {
-	    LOG_INFO << "StBFChain::Init file for geometry tag  " << geom << " has not been found in path" << path << endm;
-	  }
```

Comment:
I suppose, we might want to remove the macro files, if they are not used anymore?

Reply:
The StarDb/AgMLGeometry macros are still required.  Any maker in the chain can request the geometry through GetDatabase("VmcGeometry") at any point in the lifetime of the application (during or after StChain::Init()).  When that call is made, the DB looks for macros under StarDb/AgMLGeometry (or AgiGeometry depending on alias settings...).

The initialization of the geometry in StBFChain is only performed when certain makers are in the chain (see line 1015).